.github/actions/setup/action.yml

```
@@ -13,10 +13,10 @@ runs:
         echo '{ "auths": {} }' >> ${PFLT_DOCKERCONFIG}
       shell: bash
 
-    - name: Set up Go 1.23
+    - name: Set up Go 1.24
       uses: actions/setup-go@v4
       with:
-        go-version: 1.23.6
+        go-version: 1.24.0
```

Instead of all of these changes for ever go version, would it make sense to switch to 
```suggestion
        go-version-file: go.mod
```